internal/scheduler/scheduler.go

```diff
@@ -378,7 +378,7 @@ func (s *Scheduler) updateMetricsFromSchedulerResult(ctx *armadacontext.Context,
 		}
```

Comment:
We should remove the call to UpdatePreempted here. Since preemptions are reported when the run is actually reported as preempted.